scylla/src/frame/response/result.rs

```diff
@@ -355,14 +390,39 @@ fn deser_cql_value(typ: &ColumnType, buf: &mut &[u8]) -> StdResult<CQLValue, Par
                 )));
             }
         }),
-        Set(typ) => {
+        List(type_name) => {
+            let len: usize = types::read_int(buf)?.try_into()?;
+            let mut res = Vec::with_capacity(len);
+            for _ in 0..len {
+                // TODO: is `null` allowed as list element? Should we use read_bytes_opt?
+                let mut b = types::read_bytes(buf)?;
+                res.push(deser_cql_value(type_name, &mut b)?);
+            }
+            CQLValue::List(res)
+        }
+        Map(key_type, value_type) => {
             let len: usize = types::read_int(buf)?.try_into()?;
 
+            // let mut res_keys = Vec::with_capacity(len);
+            // let mut res_values = Vec::with_capacity(len);
+            let mut res = Vec::with_capacity(len);
+            for _ in 0..len {
+                // TODO: is `null` allowed as map element? Should we use read_bytes_opt?
```

Comment:
It's not allowed. When in doubt, just run Scylla, send the query and find out :)

Reply:
Done.